Mathlib/Topology/Order/DenselyOrdered.lean

```diff
@@ -229,6 +229,33 @@ theorem comap_coe_nhdsWithin_Iio_of_Ioo_subset (hb : s ⊆ Iio b)
     obtain ⟨x : s, hx : ∀ z, x ≤ z → z ∈ u⟩ := mem_atTop_sets.1 hu
     exact ⟨Ioo x b, Ioo_mem_nhdsWithin_Iio' (hb x.2), fun z hz => hx _ hz.1.le⟩
 
+/-- A set with a point in both the frontier and the set is not open -/
+lemma not_isOpen_of_mem_of_mem_frontier {α: Type*} [TopologicalSpace α] {s: Set α} {a: α}
```

Comment:
Since this lemma is not about orders, it should go in another file. I think it could go right below [`inter_frontier_eq`](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/Basic.html#IsOpen.inter_frontier_eq).

Also, please add a space before every colon, e.g. by writing `a : α` instead of `a: α`.